dist/controllers/bets.js

```diff
@@ -7,6 +7,11 @@ exports.updateBetStatus = exports.deleteBet = exports.updateBet = exports.create
 const express_validator_1 = require("express-validator");
 const handleErrors_1 = __importDefault(require("../utils/handleErrors"));
 const index_1 = __importDefault(require("../models/index"));
+/**
+ * Get a list of bets items from the database
```

Comment:
is it really necessary to add this comments on each controller you build?

Reply:
From my perspective, it's necessary. I know that both you and I can interpret everything about the code, but I've just envisioned a scenario where, for example, Diego needs to understand how the code flows, and this will help him.

Reply:
But I can understand that this kinds of comments on the code, just made the code to extend too much. And for the scenario that I enviosioned is not going to happen very often.

Im going to proceed to erase the comments.